README.md

```diff
@@ -2,33 +2,47 @@
 
-![ARM Tools creating web app](./images/arm-tools.gif)
+## Getting Started
 
-## Language Server
+The following guides detail how to use the ARM Tools extension features.
 
-  - Azure Resource Manager template language server for providing ARM template language completion, validation, and error guidance.
-  - Resources are validated against Azure schemas, which provide validation based on resource type and apiVersion.
-  - ARM Template Outline view for easy navigation through large templates
-  - Colorization for Template Language Expressions (TLE)
-  - Analyze and validate JSON syntax, JSON schema conformance, and string expressions.
+- [Snippets](#Snippets)
+- [Azure schema integration](#Azure-schema-completion-and-validation)
+- [Parameter file support](#parameter-files)
+- [Completions](#completions)
+- [Template navigation](#template-navigation)
+- [Other features](#other-features)
+- [Extension configuration](#extension-configuration)
 
-## Parameter files
+## Snippets
 
-Template files can be mapped to a parameter file for an enhanced authoring and validation experience. To do so use the parameter file control in the Visual Studio Code status bar, or use the right-click contextual menu on an ARM template. Once a parameter file has been mapped, the following features are available.
+Snippets are used to create basic templates and add elements such as parameters, resources, and outputs to existing templates.
 
-  - Full validation across both the template and parameter file
-  - Open an associated parameter file from a template
-  - Add all missing parameters to the parameter file and parameter auto-completion
-  - Rename parameters and all references across both the template and mapped parameter file
-  - Find and goto all references of a parameter across both the template and mapped parameter file
+### Create a new ARM template
 
-![Associate a parameters file with template and create parameters file](./images/params-support.gif)
+When in an empty JSON file, typing `arm` produces a list of scaffolding snippets. These snippets can be used to create an empty template for any ARM deployment scope (Tenant, Subscription, Management Group, and Resource Group).
 
-## Snippets
+![Image showing the arm template scaffolding snippet](./images/json-snippets.png)
+
+Notice that adding scaffolding for an ARM template changes the Visual Studio Code language mode to *Azure Resource Manager Templates*.
+
+![Image showing that the language type has changed to Azure Resource Manager Template](./images/arm-template-language.png)
+
+### Add Azure resource to an ARM template
 
-Snippets for deployment template scaffolding and 70+ Azure resources.
+Once you are working in an ARM template, placing your cursor in the resource and typing `arm` produces a list of 70+ snippets for Azure resources. Selecting a snippet adds an instance of the resource type to the ARM template.
```

Comment:
```suggestion
Once you are working in an ARM template, placing your cursor in the "resources" and typing `arm`, CTRL+SPACE or `{` produces a list of 70+ snippets for Azure resources. Selecting a snippet adds an instance of the resource type to the ARM template.
```

Similar changes needed in other places.  I'd like people to know about CTRL+SPACE (Mac or Windows) in general for completions since that seems to frequently be unknown to users.

Reply:
Fixed